4-star-nosed_mole/api_definitions/rest/ucs.yaml

```
@@ -138,10 +167,14 @@ paths:
                 $ref: "#/components/schemas/AccessURL"
         "403":
           description: |
-            The user is not registered as a Data Submitter for the
-            file corresponding to this multi-part upload.
+            Exceptions by ID:
+              - userNotRegisteredPresignPart: The user is not registered as a
+                Data Submitter for the file corresponding to this multi-part upload.
         "404":
-          description: The multi-part upload with the given ID does not exist.
+          description: |
+            Exceptions by ID:
+              - uploadNotExistingPresignPart: The multi-part upload with the given ID
```

Can be the same as above:

```suggestion
              - noSuchUpload: The multi-part upload with the given ID
```